contracts/GaslessV3.sol

```diff
@@ -97,6 +99,7 @@ contract GaslessV3 is Ownable {
         gasForSwap = _gasForSwap;
         gasForApproval = _gasForApproval;
         defaultGasPrice = _defaultGasPrice;
+        __Ownable_init();
```

Comment:
what does this do?

Reply:
it 's a initializer, necessary for upgradable contracts